docs/en/reference/second-level-cache.rst

```diff
@@ -79,23 +79,17 @@ Cache Regions
 
 ``Doctrine\ORM\Cache\Region\DefaultRegion`` It's the default implementation.
  A simplest cache region compatible with all doctrine-cache drivers but does not support locking.
-
-``Doctrine\ORM\Cache\Region`` and ``Doctrine\ORM\Cache\ConcurrentRegion``
-Defines contracts that should be implemented by a cache provider.
+``Doctrine\ORM\Cache\Region`` and ``Doctrine\ORM\Cache\ConcurrentRegion`` define contracts that should be implemented by a cache provider.
```

Comment:
I think you should change line 81 too if we are going that way, also, there should be linebreaks here.

Please comply with the sf docs standards (I don't know if we follow them, but this particular rule seems appropriate):

> [Each line should break approximately after the first word that crosses the 72nd character (so most lines end up being 72-78 characters);](http://symfony.com/doc/current/contributing/documentation/standards.html)